QoL: the last Unfettered jump drive mission is made a job (#7351)
endless-sky/endless-sky: data/hai/hai missions.txt

mission "Unfettered Jump Drive 3"
	minor

> 	landing

Reviewer: This mission doesn't fail fast enough, and shows up in the job board before jumping/reloading. (We could readd the description instead of making this mission invisible, but that still causes problems with the mission vanishing for arbitrary reasons.)
```suggestion
	landing
	invisible
```

			choice
				`	"Can you tell me what you are using the jump drives for?"`
				`	"Thank you, I will certainly bring more of them when I am able."`

> 					goto jobboard

Reviewer 2: ```suggestion
					goto job
```

			`	"Your help may indeed be beneficial to us," says the leader. "I will tell the others to contact you if they have any particular missions you can undertake."`
			choice
				`	"I look forward to hearing from them."`

> 					goto jobboard

Reviewer 2: ```suggestion
					goto job
```

				`	"What do you plan to do to the species that inhabits those worlds now?"`
			
			label wanderers
			`	"Those worlds are now held by a species of scavengers, who feast on the ruin of proud civilizations. Our scouts tell us that these carrion-feeders have wiped away nearly every Hai artifact, melting down our cities to make metal for their ships and factories, and hiding the scars of our wars beneath newly planted forests. They are an old and strong species, but few in number, and those worlds are ours by right."`
			`	You try to press them for more information, but they tell you nothing useful, aside from promising you that they will seek out your help when it is time to reclaim their territory.`

> 				accept
> 			label jobboard

Reviewer 2: Always tell the player about the job board.
```suggestion
			label job
```

> 			`	"If you want to sell more Jump Drives to us, you can do so by going to the job board."`

Reviewer 2: A little more meat.
```suggestion
			`	"If you want to provide us with more jump drives, visit the job board from now on," says the leader. "We will have warriors ready to receive them from you for the usual pay."`
```
